main.go

```diff
@@ -1,15 +1,25 @@
 package main
 
 import (
+	"flag"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/codilime/contrail-windows-docker/driver"
 	"github.com/docker/go-plugins-helpers/network"
 	"github.com/docker/go-plugins-helpers/sdk"
 )
 
 func main() {
-	d, err := driver.NewDriver()
-	if err != nil {
+	var subnet = flag.String("subnet", "172.117.0.0/16", "subnet in CIDR format for HNS")
+	var gateway = flag.String("gateway", "172.117.0.1", "default gateway IP for HNS")
+	var adapter = flag.String("adapter", "Ethernet0",
```

Comment:
In that case, i think a good idea would be to pick one of existing net adapters? There's no `Ethernet0` on my VM, so I think this driver wouldn't work there. I have default interfaces names there.